style.css

```diff
@@ -152,6 +164,253 @@ header {
   cursor: pointer;
 }
 
+/* Work section */
+
+.work-container {
+  width: 100%;
+}
+
+.work-wrapper {
+  display: grid;
+  gap: 24px;
+  grid-template-columns: 1fr 1fr;
+  padding: 24px;
+  border-radius: 24px;
+  border: 1px solid #dfe1e6;
+  width: calc(80% - 24px);
+  margin: auto;
+  margin-top: 10%;
+  background-color: var(--white-color);
+  transition: 0.3s ease-in-out;
+}
+
+.work-wrapper:hover {
+  border: 1px #a7aeff solid;
+  box-shadow: 0 50px 70px rgb(167, 174, 255, 0.24);
+}
+
+.work-img-wrapper img {
+  width: 100%;
+  height: 100%;
+}
+
+.work-desc-wrapper {
+  display: flex;
+  flex-direction: column;
+  padding: 24px;
+  gap: 20px;
+}
+
+.reversed-wrapper .work-desc-wrapper {
+  grid-row-start: 1;
+}
+
+.work-desc-wrapper h2 {
+  font-size: var(--x-medium-size);
+  font-weight: 700;
+  line-height: 52px;
+  color: var(--dark-blue);
+}
+
+.work-desc-wrapper-div {
+  display: flex;
+  align-items: center;
+}
+
+.work-desc-wrapper-div span {
+  font-size: var(--extra-small);
+  line-height: 24px;
+  font-weight: 500;
+}
+
+.work-desc-wrapper-item {
+  color: var(--lighter-blue);
+}
+
+.work-desc-wrapper-position {
+  color: var(--light-blue);
+}
+
+.work-desc-wrapper-div i {
+  width: 8px;
+  height: 8px;
+  border-radius: 50%;
+  margin: 0 10px;
+  background-color: var(--light-gray);
+}
+
+.work-desc-wrapper p {
+  color: var(--light-blue);
+  font-size: var(--small-size);
+  font-weight: 400;
+  line-height: 24px;
+}
+
+.work-desc-wrapper ul {
+  display: flex;
+  gap: 10px;
+}
+
+.work-desc-wrapper ul li {
+  list-style: none;
+  background-color: var(--lighter-gray);
+  padding: 4px 12px;
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: var(--xx-small-size);
+  color: var(--menu-blue);
+}
+
+.work-desc-wrapper button,
+.about-media-link button {
+  height: 48px;
+  width: fit-content;
+  padding: 12px;
+  margin-top: 10px;
+  outline: none;
+  border-radius: 8px;
+  border: 1px var(--lightest-blue) solid;
+  font-family: var(--poppins);
+  font-size: calc(var(--extra-small) - 1px);
+  font-weight: 500;
+  background-color: var(--white-color);
+  color: var(--lightest-blue);
+  cursor: pointer;
+  transition: 0.3s ease-in-out;
+}
```

Comment:
- [OPTIONAL]  kindly make sure you're using the same font style and font-weight as the [Figma](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/images/m2_mobile/works-mobile.png) template